### DISCUSSION TITLE
Make the engines push to limits And play anti-drawing moves. How?

### DISCUSSION THREAD

**Discussion by olypnsf** [lichess-bot-devs/lichess-bot#807] [General]
I tried to make the engine push to their limits and play anti-drawing moves. Since my bot @GeometrisBot is not playing the best moves as possible as 2900 rating. What the changes of config.yml I should do here?. Hope you can answer

**Comment by AttackingOrDefending** [comment]
Did you disable `go_commands` and lower `Move Overhead`? These are the most important changes.

**Comment by AttackingOrDefending** [comment]
Closing since the issue was also closed.
